src/main/java/org/opensearch/sdk/ActionExtension.java

```diff
@@ -69,7 +69,7 @@ default List<ActionFilter> getActionFilters() {
      *
      * @return a list of REST handlers (REST actions) this extension handles.
      */
-    default List<ExtensionRestHandler> getExtensionRestHandlers() {
+    default List<ExtensionRestHandler> getExtensionRestHandlers(ExtensionsRunner extensionsRunner) {
```

Comment:
I really don't like the idea of changing the method signature of an existing extension point.